components/auto-complete/auto-complete-dropdown.tsx

```diff
@@ -21,35 +24,61 @@ export type AutoCompleteDropdownProps = Props & typeof defaultProps & NativeAttr
 
 const AutoCompleteDropdown: React.FC<React.PropsWithChildren<AutoCompleteDropdownProps>> = ({
   children,
+  variant,
   visible,
   className,
   dropdownStyle,
   disableMatchWidth,
 }) => {
+  const isSolid = useMemo(() => variant === 'solid', [variant])
   const theme = useTheme()
-  const { ref } = useAutoCompleteContext()
+  const { ref, value } = useAutoCompleteContext()
+  const { border, hoverBorderColor, hoverBackgroundColor } = useMemo(
+    () => getColors(theme.palette, 'default', isSolid),
+    [theme.palette, status, isSolid],
+  )
+
   const clickHandler = (event: React.MouseEvent<HTMLDivElement>) => {
     event.preventDefault()
     event.stopPropagation()
     event.nativeEvent.stopImmediatePropagation()
   }
 
   return (
-    <Dropdown parent={ref} visible={visible} disableMatchWidth={disableMatchWidth}>
+    <Dropdown
+      parent={ref}
+      visible={visible}
+      disableMatchWidth={disableMatchWidth}
+      clearTime={value === '' ? 0 : 60}
+      leaveTime={value === '' ? 0 : 60}>
       <div
-        className={`auto-complete-dropdown ${className}`}
+        className={`auto-complete-dropdown ${className} ${isSolid ? 'solid' : 'line'}`}
```

Comment:
Should increase custom className's priority

Reply:
You mean put the `${className}` after the solid/line?

Reply:
yes

Reply:
Done